### PR TITLE
removed background image library text if collection is empty

### PR DESCRIPTION
**Pull request by alexey-petrov**
Closes https://github.com/sparkletown/internal-sparkle-issues/issues/716

Done as per https://github.com/sparkletown/internal-sparkle-issues/issues/716#issuecomment-839728182

'Select background image' info text won't appear unless background library collection is not empty

### CODE REVIEW

File: src/components/molecules/ImageInput/ImageCollectionInput.tsx [hunk 1]
```diff
@@ -131,9 +131,11 @@ export const ImageCollectionInput: React.FC<ImageInputProps> = (props) => {
...
       />
       {error?.message && <span className="input-error">{error.message}</span>}
-      <div style={{ marginTop: 10, fontSize: "16px" }}>
-        {`Or choose one of our popular ${imageType}`}
-      </div>
+      {imageCollection.length ? (
+        <div style={{ marginTop: 10, fontSize: "16px" }}>
+          {`Or choose one of our popular ${imageType}`}
+        </div>
+      ) : null}
```
Comment (mike-lvov) [inline src/components/molecules/ImageInput/ImageCollectionInput.tsx:138]: Please, move the length to `hasImageCollections` and then use hasImageCollections && <div>..., not a ternary

**Review by mike-lvov — CHANGES_REQUESTED** [review]

File: src/components/molecules/ImageInput/ImageCollectionInput.tsx (hunk 1 above)
Comment (alexey-petrov) [reply src/components/molecules/ImageInput/ImageCollectionInput.tsx]: fixed in 65b0fee

**Review by mike-lvov — APPROVED** [review]
LGTM

**Comment by whoaitsaimz** [comment]
Also not assigned to correct Epic.

**Pull request merged**